cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1003,6 +1026,18 @@ def make_resource(self, data, **kwargs):
         return AwsBatchComputeResource(**data)
 
 
+class ByosComputeResourceSchema(SlurmComputeResourceSchema):
+    """Represent the schema of the Byos ComputeResource."""
+
+    instance_types = fields.List(fields.Str(), required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
```

Comment:
there are a series of missing changes to support instance_types. Also you are allowing both `instance_types` and `instance_type`.
For now let's leave only instance_type and we'll introduce instance_types at a later stage.